src/pages/Admin/Venue/AnnouncementOptions/AnnouoncementStatus/AnnouncementStatus.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import classNames from "classnames";
+
+import { BannerFormData } from "types/banner";
+
+import iconCheck from "assets/icons/icon-checkmark.svg";
+import iconClose from "assets/icons/icon-close.svg";
+
```

Comment:
As called out before, can we reuse `@fortawesome` here?

Reply:
https://github.com/sparkletown/sparkle/pull/1440/commits/c7176f5f7229863de63943d302f579e2424e3b20